core/templates/pages/contributor-dashboard-admin-page/contributor-dashboard-tables/contributor-admin-stats-table.component.spec.ts

```diff
@@ -313,4 +334,195 @@ describe('Contributor stats component', () => {
         } as QuestionReviewerStatsData));
     }));
   });
+
+  describe('when toggled various model', () => {
+    beforeEach(() => {
+    });
+
+    it('should open question role editor modal and return not' +
+      ' translation submitter', fakeAsync(() => {
```

Comment:
Not really sure what this means. Do you mean the question role editor modal should be opened when on the question submitter tab? Ditto below. The test name should specify the test condition.

Reply:
Updated. PTAL again if that seems good.